packages/core/src/UserState.ts

```diff
@@ -237,6 +237,23 @@ export default class UserState {
                 epoch: signup.epoch,
             })
         }
+        const allNullifiers = [] as any
+        for (let x = signup?.epoch ?? 0; x <= toEpoch; x++) {
+            allNullifiers.push(
+                ...[0, this.sync.settings.numEpochKeyNoncePerEpoch].map((v) =>
+                    genEpochKey(this.id.secret, attesterId, x, v).toString()
+                )
+            )
+        }
+        const foundNullifiers = await this.sync._db.findMany('Nullifier', {
+            where: {
+                attesterId,
+                nullifier: allNullifiers,
```

Comment:
Is it an `OR` logic of `allNullifiers`?

Reply:
yes, it selects documents where `nullifier` is a value in `allNullifiers`